sql/updates/world/master/2024_99_99_99_ragefire_chasm_instance_smart_ai.sql

```diff
@@ -0,0 +1,608 @@
+-- Ragefire Chasm Quests and SAI for Scouts/Rangers and Misc
+UPDATE `creature_template_difficulty` SET `ContentTuningID`=207, `StaticFlags1`=0 WHERE `Entry`=61788; -- SI:7 Ranger
+UPDATE `creature_template_difficulty` SET `ContentTuningID`=207, `StaticFlags1`=0 WHERE `Entry`=61680; -- Kor'kron Scout
+UPDATE `creature_template` SET `unit_flags`=32832, `unit_flags2`=2048 WHERE `entry`=61680; -- Kor'kron Scout
+UPDATE `creature_template` SET `unit_flags`=32832, `unit_flags2`=2048 WHERE `entry`=61788; -- SI:7 Ranger
```

Comment:
Current parser doesn't update unit_flags adding UNIT_FLAG_CAN_SWIM (0x8000) (it should update StaticFlags1 adding CREATURE_STATIC_FLAG_CAN_SWIM)